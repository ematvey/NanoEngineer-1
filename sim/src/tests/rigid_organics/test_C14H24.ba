LENGTH 0 1 1.53250448613
LENGTH 0 2 1.52858692916
LENGTH 0 14 1.09594434165
LENGTH 0 15 1.09424768677
LENGTH 1 3 1.53216774539
LENGTH 1 4 1.5217558937
LENGTH 1 5 1.53701138577
LENGTH 2 6 1.52358819896
LENGTH 2 7 1.53779940174
LENGTH 2 8 1.53343438073
LENGTH 3 9 1.52939367071
LENGTH 3 16 1.09646614175
LENGTH 3 17 1.09387796394
LENGTH 4 18 1.09189926275
LENGTH 4 19 1.08994954012
LENGTH 4 20 1.08995871481
LENGTH 5 12 1.52828433218
LENGTH 5 21 1.09632203298
LENGTH 5 22 1.08967976947
LENGTH 6 23 1.08239410568
LENGTH 6 24 1.0879342811
LENGTH 6 25 1.09299451051
LENGTH 7 12 1.53334862311
LENGTH 7 26 1.09555237209
LENGTH 7 27 1.09358218713
LENGTH 8 9 1.53200848562
LENGTH 8 28 1.09053977461
LENGTH 8 29 1.086547744
LENGTH 9 10 1.52561004192
LENGTH 9 11 1.53984349854
LENGTH 10 30 1.09425134224
LENGTH 10 31 1.08859037291
LENGTH 10 32 1.08827615981
LENGTH 11 12 1.53520878059
LENGTH 11 33 1.09389670445
LENGTH 11 34 1.09091612877
LENGTH 12 13 1.52889175549
LENGTH 13 35 1.08384408473
LENGTH 13 36 1.09279458271
LENGTH 13 37 1.08733711424
ANGLE 0 1 3 108.366367501
ANGLE 0 1 4 110.554780966
ANGLE 0 1 5 108.139692405
ANGLE 0 2 6 111.053863159
ANGLE 0 2 7 108.118857122
ANGLE 0 2 8 108.574476379
ANGLE 1 0 2 111.814441998
ANGLE 1 0 14 109.51245535
ANGLE 1 0 15 110.004687395
ANGLE 1 3 9 111.636934141
ANGLE 1 3 16 109.84134417
ANGLE 1 3 17 109.743100626
ANGLE 1 4 18 111.367976052
ANGLE 1 4 19 111.138620762
ANGLE 1 4 20 111.271748686
ANGLE 1 5 12 111.28199817
ANGLE 1 5 21 110.022715553
ANGLE 1 5 22 109.466743019
ANGLE 2 0 14 109.237574675
ANGLE 2 0 15 109.846064863
ANGLE 2 6 23 112.177153378
ANGLE 2 6 24 111.012709473
ANGLE 2 6 25 111.079601577
ANGLE 2 7 12 111.450087754
ANGLE 2 7 26 109.823198117
ANGLE 2 7 27 109.717136732
ANGLE 2 8 9 111.549097037
ANGLE 2 8 28 109.807513959
ANGLE 2 8 29 109.931554678
ANGLE 3 1 4 110.730377313
ANGLE 3 1 5 108.738108277
ANGLE 3 9 8 108.598304306
ANGLE 3 9 10 110.538561399
ANGLE 3 9 11 108.061115465
ANGLE 4 1 5 110.237764905
ANGLE 5 12 7 108.608180296
ANGLE 5 12 11 108.491490596
ANGLE 5 12 13 110.413449627
ANGLE 6 2 7 110.207521157
ANGLE 6 2 8 110.426357503
ANGLE 7 2 8 108.379015244
ANGLE 7 12 11 108.493957827
ANGLE 7 12 13 110.50541189
ANGLE 8 9 10 110.437296844
ANGLE 8 9 11 108.490713696
ANGLE 9 3 16 109.599845622
ANGLE 9 3 17 109.440286011
ANGLE 9 8 28 109.994540788
ANGLE 9 8 29 109.388485666
ANGLE 9 10 30 110.929771542
ANGLE 9 10 31 111.528027884
ANGLE 9 10 32 111.245651147
ANGLE 9 11 12 111.453456021
ANGLE 9 11 33 109.356150866
ANGLE 9 11 34 109.639143111
ANGLE 10 9 11 110.639149523
ANGLE 11 12 13 110.26769646
ANGLE 12 5 21 109.506941764
ANGLE 12 5 22 109.647371334
ANGLE 12 7 26 109.58439912
ANGLE 12 7 27 109.512338963
ANGLE 12 11 33 109.591710724
ANGLE 12 11 34 109.918584574
ANGLE 12 13 35 111.344152378
ANGLE 12 13 36 110.469245422
ANGLE 12 13 37 111.245284615
ANGLE 14 0 15 106.267686076
ANGLE 16 3 17 106.441714623
ANGLE 18 4 19 107.809779352
ANGLE 18 4 20 107.755426032
ANGLE 19 4 20 107.309428657
ANGLE 21 5 22 106.810274141
ANGLE 23 6 24 107.285842654
ANGLE 23 6 25 107.480184301
ANGLE 24 6 25 107.584840104
ANGLE 26 7 27 106.63391025
ANGLE 28 8 29 106.018200865
ANGLE 30 10 31 107.843216119
ANGLE 30 10 32 107.342780253
ANGLE 31 10 32 107.766720093
ANGLE 33 11 34 106.766201445
ANGLE 35 13 36 107.470804353
ANGLE 35 13 37 108.337265737
ANGLE 36 13 37 107.822477042
