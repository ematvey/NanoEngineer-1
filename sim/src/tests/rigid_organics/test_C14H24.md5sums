stdout 127B74B2E50E0499FF67A0B012B6A8DC
stderr D41D8CD98F00B204E9800998ECF8427E
