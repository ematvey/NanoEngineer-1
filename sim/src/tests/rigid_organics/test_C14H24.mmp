mmpformat 050502 required; 050706 preferred
kelvin 300
group (View Data)
info opengroup open = True
csys (HomeView) (1.000000, 0.000000, 0.000000, 0.000000) (10.000000) (0.000000, 0.000000, 0.000000) (1.000000)
csys (LastView) (1.000000, 0.000000, 0.000000, 0.000000) (8.645131) (-0.000000, -0.000000, -0.000000) (1.000000)
egroup (View Data)
group (C14H24)
info opengroup open = True
mol (C14H24.pdb) def
atom 1 (6) (-431, 51, 1577) def
atom 2 (6) (-553, -1682, 957) def
bond1 1
atom 3 (6) (188, 1190, 239) def
bond1 1
atom 4 (6) (893, -1342, 39) def
bond1 2
atom 5 (6) (-331, -1721, 2157) def
bond1 2
atom 6 (6) (-1875, -1692, 396) def
bond1 2
atom 7 (6) (-93, 2264, 239) def
bond1 3
atom 8 (6) (-1240, 826, -783) def
bond1 3
atom 9 (6) (545, 463, -1089) def
bond1 3
atom 10 (6) (1514, -1015, -1685) def
bond1 4 9
atom 11 (6) (2519, -1544, -1031) def
bond1 10
atom 12 (6) (845, -1354, -1833) def
bond1 10
atom 13 (6) (-1794, -389, -1325) def
bond1 6 8 12
atom 14 (6) (-2411, -822, -2647) def
bond1 13
atom 15 (1) (-1323, 1010, 1790) def
bond1 1
atom 16 (1) (448, 796, 1792) def
bond1 1
atom 17 (1) (984, -2948, -34) def
bond1 4
atom 18 (1) (1672, -1695, 1103) def
bond1 4
atom 19 (1) (-559, -2938, 1942) def
bond1 5
atom 20 (1) (-784, -1157, 2576) def
bond1 5
atom 21 (1) (39, -1346, 3344) def
bond1 5
atom 22 (1) (-1629, -2867, 22) def
bond1 6
atom 23 (1) (-2671, -1131, 747) def
bond1 6
atom 24 (1) (-125, 3228, -579) def
bond1 7
atom 25 (1) (284, 2943, 924) def
bond1 7
atom 26 (1) (-1512, 2523, -424) def
bond1 7
atom 27 (1) (-1435, 1050, -1892) def
bond1 8
atom 28 (1) (-2322, 721, -623) def
bond1 8
atom 29 (1) (1215, 1152, -1840) def
bond1 9
atom 30 (1) (2510, 736, 441) def
bond1 9
atom 31 (1) (2674, -1096, -2260) def
bond1 11
atom 32 (1) (2704, -2130, -1660) def
bond1 11
atom 33 (1) (3212, -1050, -348) def
bond1 11
atom 34 (1) (205, -1708, -2345) def
bond1 12
atom 35 (1) (212, -2440, -1659) def
bond1 12
atom 36 (1) (-2193, -610, -3214) def
bond1 14
atom 37 (1) (-3121, -1060, -1772) def
bond1 14
atom 38 (1) (-2820, -2489, -2492) def
bond1 14
egroup (C14H24)
end1
group (Clipboard)
info opengroup open = False
egroup (Clipboard)
end molecular machine part C14H24

