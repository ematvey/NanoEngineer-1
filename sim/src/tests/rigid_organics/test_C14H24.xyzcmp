38
C14H24.mmp
C -0.400000 0.274000 1.261000
C -0.388000 -1.251000 1.110000
C -0.172000 0.979000 -0.076000
C 0.963000 -1.667000 0.519000
C -0.617000 -1.923000 2.456000
C -1.500000 -1.643000 0.124000
C -0.187000 2.493000 0.094000
C -1.285000 0.542000 -1.043000
C 1.174000 0.518000 -0.648000
C 1.215000 -1.002000 -0.835000
C 2.565000 -1.433000 -1.400000
C 0.074000 -1.408000 -1.786000
C -1.294000 -0.979000 -1.237000
C -2.405000 -1.389000 -2.204000
H -1.362000 0.593000 1.678000
H 0.364000 0.585000 1.980000
H 1.000000 -2.757000 0.406000
H 1.767000 -1.400000 1.211000
H -0.597000 -3.011000 2.366000
H -1.582000 -1.641000 2.877000
H 0.148000 -1.635000 3.177000
H -1.525000 -2.732000 0.000000
H -2.467000 -1.350000 0.532000
H -0.034000 3.011000 -0.844000
H 0.598000 2.817000 0.774000
H -1.141000 2.833000 0.505000
H -1.150000 1.035000 -2.012000
H -2.256000 0.867000 -0.659000
H 1.361000 1.016000 -1.600000
H 1.981000 0.822000 0.013000
H 2.754000 -0.957000 -2.367000
H 2.612000 -2.511000 -1.544000
H 3.379000 -1.151000 -0.735000
H 0.237000 -0.948000 -2.765000
H 0.091000 -2.488000 -1.939000
H -2.270000 -0.931000 -3.177000
H -3.379000 -1.071000 -1.824000
H -2.435000 -2.468000 -2.335000
