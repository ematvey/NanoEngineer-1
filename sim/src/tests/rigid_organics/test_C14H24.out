======= test_C14H24.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
1
======= lengthsangles =======
OK
======= stderr =======
cutover to fine tolerance at 3
== structure comparison ==
cutover to fine tolerance at 1
